lib/travis/build/addons/browserstack.rb

```diff
@@ -0,0 +1,156 @@
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class BrowserStack < Base
```

Comment:
Since the class name is defined as `BrowserStack`, the key we use in `.travis.yml` is `browser_stack`:

```yaml
addons:
  browser_stack:
    access_key: 88374hd417drQ23
```